packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -98,6 +105,7 @@ export default function useFeedSettings(): FeedSettingsReturnType {
 
       return { ...req, feedSettings };
     },
+    { enabled: isTesting || !observerOnly },
```

Comment:
Do we have cases where we set the value but not watch in the same hook?
Just not really sure what the use-case for not having it is?

Reply:
This is when we open the feed filters popup it should make a graphql request - now when we open the `Tags` section, it also uses the same hook, which since it is a different instance, it would again make the same graphql request, which it should not because the data was already available.

What do you think about this? I am also having some doubts if the reduction of calls would be worth it since we are not yet experiencing any issues with the number of requests.

Reply:
My main concern was if it would cause any "negative" effects.
I wouldn't see any, and like you said we don't have any issues with the performance yet.

But maybe @idoshamun has an idea on this?

Reply:
Probably this should be a subject on a different PR if in any case. I can revert it for now so we can proceed with the primary concern.